client/state/sites/reducer.js

```diff
@@ -75,6 +75,15 @@ export function items( state = {}, action ) {
 			}
 			return state;
 
+		case 'PRESSABLE_TEST':
```

Comment:
The name of this would more likely be something closer to `PRESSIBLE_TRANSFER_START`. Do you mind adding a new action type, even though we might rename this later.